src/components/Other/Projects.vue

```diff
@@ -48,8 +48,7 @@
       <div class="text-xs-center">
         <v-dialog
           v-model="dialog"
-          full-width
-          scrollable
+          width="500"
```

Comment:
Use max-width instead

Reply:
max-width will increase the size of the dialog box

Reply:
OK